projects/packages/jetpack-mu-wpcom/src/features/help-center/class-help-center.php

```diff
@@ -171,6 +172,22 @@ function ( $wp_admin_bar ) {
 				'before'
 			);
 		}
+
+		if ( ! is_admin() ) {
+			$stylesheet     = is_rtl() ? 'build/components/style-rtl.css' : 'build/components/style.css';
+			$stylesheet_url = plugins_url( 'gutenberg/' . $stylesheet );
+			if ( function_exists( 'gutenberg_url' ) ) {
+				// @phan-suppress-next-line PhanUndeclaredFunction
+				$stylesheet_url = gutenberg_url( $stylesheet );
+			}
+			// Enqueue wp-component styles because they're not enqueued in wp-admin outside of the editor.
+			wp_enqueue_style(
+				'wp-components-help',
```

Comment:
I'm not sure if this is a good idea. It will duplicate the styles if they're already enqueued. It's great for debugging, but if it works when you do this, I think that means dashicons are missing.

Reply:
Good point, I've updated the handle to use the standard `wp-components` and to include the `dashicons` dependencies and it works ok on Atomic.